Worktrolly/worktrolly
# Added completion date and made patch for Completion Date in older tasks

Signed-off-by: Arjun Dave <davearjun25@gmail.com>

### Functionality:
Shows completion date when the user submits Status as "Completed" while logging a task and older tasks which are completed have completion date as "XX/XX/XXXX".

### Solution:
Changed the firebase functions,
Added lines to create a CompletionDate field with value "Not yet Completed" while creating a new task and in log work added lines to update the same field to the current date on logging task as Completed.

### Risk level:

### How to test:
1. First, deploy the functions file in the code.
2. Use the patch to mark older Completed tasks with the date "XX/XX/XXXX"
3. ng serve
4. Create a new task and then log work and mark task as completed and voila Completion Date appears in the Task Detail page.

### Pictures of the updated Task Detail page:

#### When the task isn't completed,

![image](https://user-images.githubusercontent.com/58979519/106028568-1d9b4800-60f2-11eb-96fd-746f9a9e4ccc.png)

#### After Completing the task,

![image](https://user-images.githubusercontent.com/58979519/106028774-60f5b680-60f2-11eb-9457-6625f5b619fa.png)
